src/OptionsPopup.qml

```diff
@@ -488,7 +488,12 @@ Popup {
             }
             chkWifi.checked = true
         } else {
-            fieldWifiCountry.currentIndex = fieldWifiCountry.find("GB")
+            var language_country = Qt.locale().name
+            var country = language_country.split('_')[1]
+            var index = fieldWifiCountry.find(country)
+            if (index < 0)
+                index = fieldWifiCountry.find(qsTr("GB"))
```

Comment:
In case language_country doesn't work for a locale, a ts file (e.g. language_country.ts) can decide which country should be selected by default. it's convenient, isn't it?

Reply:
Ahh, it wasn't obvious that that was why you were doing this. That's not _really_ what the ts files are supposed to be used for, so perhaps that ought to be done in a separate PR? :shrug: 

(and just to clarify - I don't work on this repo myself, so please don't apply too much weight to my comments!)

Reply:
This is actually what the ts files are supporsed to be used for.

https://doc.qt.io/qt-5/i18n-source-translation.html#using-tr-for-all-literal-text says
> Wherever your program uses a string literal (quoted text) that will be presented to the user

"GB" here is (not directly but meant to be) presented to the user.

Reply:
Ah, I dug into it a bit more and you're correct - the string "GB" _does_ get shown to the user, as it's one of the entries in the `fieldWifiCountry` ComboBox. However, the entries in that combo-box come from [this](https://github.com/raspberrypi/rpi-imager/blob/qml/src/imagewriter.cpp#L828) function (which in turn reads from [this](https://github.com/raspberrypi/rpi-imager/blob/qml/src/countries.txt) file), and as the entries there don't get localised, then the "GB" here probably shouldn't be localised either? :man_shrugging:

Reply:
The default wifi country should be localized. I hope one from locale would work. [this](https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/text/qlocale_data_p.h?h=5.15.2#n6670) is countries a locale could provide. In case not, how we can provide better default country than "GB" that doesn't work for most of users?
IMHO, the .ts files seems the right way for fallback.

Reply:
I think that if it was a human-language string like "Great Britain", then it _would_ make sense to mark it as localisable with `qsTr()`; but given that this "GB" is a machine-readable string (which gets inserted directly into a config-file) then it _doesn't_ make sense to mark it as localisable. All IMHO of course.
(e.g. a Croatian speaker might translate "GB" (Great Britain) to "VB" (Velika Britanija), but that wouldn't work as `VB` isn't a valid country-code)

> In case not, how we can provide better default country than "GB" that doesn't work for most of users?

I think the "proper" way of doing that would probably be to hook into https://github.com/raspberrypi/rpi-imager/blob/qml/src/imagewriter.cpp#L173 and https://github.com/raspberrypi/rpi-imager/blob/qml/src/imagewriter.cpp#L1142 ? :shrug: